### ISSUE TITLE
Object N

### ISSUE THREAD

**Issue by JoshuaFern** [luxtorpeda-dev/packages#422]
### Project Link

https://github.com/KleskBY/ObjectN-DarkPlaces-engine

### Project License

GPL-3.0

### Steam App Ids

- https://steamdb.info/app/1508030/

### Additional Context

Free to play game, looks to run on a Quake engine.

**Comment by d10sfan** [comment]
This one has a strange decision to pre-build SDL2 and curl, which is causing issues with the build. I've been able to build many quake-like engines, but haven't had success with this one yet.

@JoshuaFern Have you been able to build this one?

**Comment by JoshuaFern** [comment]
I haven't attempted a build, however the best solution might just be to launch the game with another quake engine instead of building this one-off fork.

**Comment by d10sfan** [comment]
@JoshuaFern I tried darkplaces and it somewhat worked, although there were issues (multiple warnings and the hud has alot of broken elements).

You can try it if you want by doing this:

* Copy darkplaces-2017 from Quake to this game directory
* ```./darkplaces-2017/darkplaces-sdl -basedir ./ -game data```

vkquake and fte quake world didn't want to launch it at all.

**Comment by JoshuaFern** [comment]
Ah, I see what you mean. This fork is a significant divergence from the original darkplaces engine, with over 295,241 additions according to the github stats. It's not a simple matter of just playing it with the original engine as I had hoped.

**Comment by d10sfan** [comment]
@JoshuaFern I was able to figure out that the issue seems related to the "nuklear" menu that was implemented. I created a patch that removes it and it compiles now and seems to basically function (the in-game menu is also functional as far as I can tell).

I've released that in https://github.com/luxtorpeda-dev/packages/pull/423, although if anyone is able to figure out how to make the menu compilable as well, that would help, but at least it compiles and runs well.
